config/grafana/dashboard.json

```
@@ -1999,10 +1999,27 @@
     "tags": [],
     "templating": {
       "list": [
+      {
+          "current": {
+            "selected": false,
+            "text": "Prometheus",
+            "value": "Prometheus"
+          },
+          "hide": 0,
```

```suggestion
          "hide": 2,
```
So the variable is hidden and the default Prometheus data source is selected

---

I'd think it's better to have this variable visible. For example, one may want to switch between prometheus (with short term storage) and Thanos.

---

Ok to me. :)